Kernel/Config/Files/XML/Framework.xml

```diff
@@ -9217,17 +9217,59 @@ via the Preferences button after logging in.
            </Hash>
         </Value>
     </Setting>
-    <!-- Special settings are possible for the indices Ticket, Customer, CustomerUser and ConfigItem
-         If not set the general settings defined al 'all' are used. -->
-    <Setting Name="Elasticsearch::ArticleIndexCreationSettings###Ticket" Required="0" Valid="1">
-        <Description Translatable="1">Number of shards (NS) and replicas (NR) for the index ticket.</Description>
+    <Setting Name="Elasticsearch::IndexSettings" Required="0" Valid="0">
```

Comment:
Be careful here. This is technically not wrong, but I would prefer, if you used Elasticsearch::IndexSettings###Default. At the moment $ConfigObject->Get('Elasticsearch::IndexSettings') returns the perl structure:
`NS => 1,
NR => 0,
FieldsLimit => 2000,
Ticket => {
    NS => 1,
    ...,
},
Customer => {...},
...,`
While this can be used, as we want people to be flexible with adding more stuff to the index configuration and thus maybe ConfigOptions themselves, too, I would rather have a strict separation there.

Reply:
ok, point taken.